### PR TITLE
RS-251: add product navigation in main menu

### PR DESCRIPTION
**Pull request by AnthonyCvn**
- New navigation menu
- Show only recent blogs on sidebar
- Add "Schedule a Technical Review" to contact form
- Use "Subject=..." parameter in URL to pre-select reason in contact form
- Remove social bar under blogs. See bug when clicking on tags, e.g. https://www.reduct.store/blog/tags/computer-vision: 
Screenshot 2024-04-12 at 18 35 14

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -110,18 +110,109 @@ const config = {
...
+                to: "/blog/comparisons/iot/reductstore-vs-mongodb",
+              },
+              {
+                label: "Schedule a Technical Review",
+                to: "/contact?subject=TechnicalReview",
+              },
+            ],
+          },
+          {
+            type: "dropdown",
```
Comment (atimin) [inline docusaurus.config.js:144]: Maybe we can add it  when we will have use case pages? We discussed  them like blog posts but I believe they should have different structure.

File: docusaurus.config.js [hunk 2]
```diff
@@ -110,18 +110,109 @@ const config = {
...
+              {
+                label: "vs TimescaleDB",
+                to: "/blog/comparisons/iot/reductstore-vs-timescaledb",
+              },
+              {
+                label: "vs MongoDB",
+                to: "/blog/comparisons/iot/reductstore-vs-mongodb",
+              },
+              {
+                label: "Schedule a Technical Review",
```
Comment (atimin) [inline docusaurus.config.js:138]: I'm not sure it is about product.

**Review by atimin — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (AnthonyCvn) [reply docusaurus.config.js:144]: @atimin I agree, although, don't you think it's better than nothing while waiting for the use cases?

File: docusaurus.config.js (hunk 2 above)
Comment (AnthonyCvn) [reply docusaurus.config.js:138]: @atimin checkout TimescaleDB, they added this button on their product tab. Could be good to get more people to send us an email. Or maybe a button somewhere else?

File: docusaurus.config.js [hunk 3]
```diff
@@ -110,18 +110,109 @@ const config = {
...
+              {
+                label: "API Reference",
+                to: "/docs/http-api",
+              },
+              {
+                label: "Download",
+                to: "/download",
+              },
+              {
+                label: "GitHub",
```
[the remark this answers is not in the document]
Comment (AnthonyCvn) [reply docusaurus.config.js:201]: Will do :)

File: docusaurus.config.js (hunk 2 above)
Comment (atimin) [reply docusaurus.config.js:138]: Agree, don't have a better idea so far. Let's keep it here for now.

File: docusaurus.config.js (hunk 1 above)
Comment (atimin) [reply docusaurus.config.js:144]: Okay, let's keep it and focus on the use cases.

**Pull request merged**